src/hoi4_world/localizations/localizations_converter_tests.cpp

```diff
@@ -499,4 +521,132 @@ TEST(Hoi4worldLocalizationsLocalizationsconverter, CharacterLocalizationsCanBeCo
            testing::Pair("spanish", "$Doug$ $Romney1$")));
 }
 
+
+TEST(Hoi4worldLocalizationsLocalizationsconverter, IdeaLocalizationsCanBeConverted)
+{
+   commonItems::LocalizationDatabase vic3_localizations("english", {"spanish"});
+   commonItems::LocalizationBlock block_one("male", "english");
+   block_one.ModifyLocalization("english", "MaleName");
+   block_one.ModifyLocalization("spanish", "MaleNombre");
+   vic3_localizations.AddOrModifyLocalizationBlock("male", block_one);
+   commonItems::LocalizationBlock block_two("female", "english");
+   block_two.ModifyLocalization("english", "FemaleName");
+   block_two.ModifyLocalization("spanish", "FemaleNombre");
+   vic3_localizations.AddOrModifyLocalizationBlock("female", block_two);
+   commonItems::LocalizationBlock block_three("monarch", "english");
+   block_three.ModifyLocalization("english", "Surname");
+   block_three.ModifyLocalization("spanish", "Surnombre");
+   vic3_localizations.AddOrModifyLocalizationBlock("monarch", block_three);
+
+
+   const Character character_one({.first_name = "male", .last_name = "monarch", .is_female = false});
+   const Character character_two({.first_name = "female", .last_name = "monarch", .is_female = true});
+
+   const Localizations hoi4_localizations = ConvertLocalizations(vic3_localizations,
+       {},
+       {},
+       {},
+       mappers::ProvinceMapper({}, {}),
+       {},
+       {},
+       {
+           {"ONE", Country({.monarch_idea_id = 1})},
+           {"TWO", Country({.monarch_idea_id = 2})},
+           {"NON", Country({.monarch_idea_id = std::nullopt})},
+           {"MIS", Country({.monarch_idea_id = 3})},
+       },
+       {
+           {1, character_one},
+           {2, character_two},
+       });
+
+   EXPECT_EQ(hoi4_localizations.GetIdeaLocalizations().size(), 4);
+
+   const std::string idea_one_name = hoi4::GetMonarchIdeaName("ONE", character_one);
+   const std::string idea_one_desc = fmt::format("{}_desc", idea_one_name);
+   const std::optional<commonItems::LocalizationBlock> one_localization_block =
+       hoi4_localizations.GetIdeaLocalizations().GetLocalizationBlock(idea_one_name);
+   EXPECT_THAT(one_localization_block.value_or(commonItems::LocalizationBlock{"", ""}).GetLocalizations(),
+       testing::UnorderedElementsAre(testing::Pair("english", "King MaleName Surname"),
```

Comment:
Marginally relevant, not all monarchs are kings/queens. Title should be somehow related with actual Vic3 title for monarchs, ranging from RULER_TITLE_EMPEROR to RULER_TITLE_GRAND_PRINCESS, depending on a host of factors... Ideally importing Vic3 localization for title.

Reply:
Or, for now, just ditch "King" and put unisex "Ruler" maybe...

Reply:
Opened #501 to fix this in the future.